packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -109,9 +117,20 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         sharedObjectRegistry: ISharedObjectRegistry,
         existing: boolean,
     ): FluidDataStoreRuntime {
-        return new FluidDataStoreRuntime(context, sharedObjectRegistry, existing);
+        return new FluidDataStoreRuntime(
+            context,
+            sharedObjectRegistry,
+            existing,
+            async (rt) => requestFluidObject(rt, "/"));
```

Comment:
This does not feel right. In a sense that why "/"?  Why not return handle.get()? here?

Reply:
Maybe it's fine, if it's deprecated, though would be nice to remove usage from our unit tests.
I think as a general rule, we should strive to remove usage of deprecated APIs from our own repo when we deprecate them, as a way to prove to ourselves it's doable, and demonstrate to other right patterns.

Reply:
For some reason "find references" was not giving me results when I first tried it but I see them now. Agreed, I'll replace those with the new way of doing things.

The intention behind line 124 there was "if someone is still using the load() method, and thus not providing an entrypoint initialization function (which we're not adding to it to avoid breaking changes for now), let's default it to the current pattern of requesting the runtime's root object, but that way at least we'll have a working entrypoint".